src/video_info/player_response/video_details.rs

```diff
@@ -36,6 +40,13 @@ pub struct VideoDetails {
     pub view_count: u64,
 }
 
+#[derive(Clone, Debug, Deserialize, Serialize, PartialEq, Eq, Hash)]
+#[serde(rename_all = "SCREAMING_SNAKE_CASE")]
+pub enum LatencyClass {
+    #[serde(rename = "MDE_STREAM_OPTIMIZATIONS_RENDERER_LATENCY_LOW")]
+    LatencyLow,
```

Comment:
There seems to be some duplication in the name `LatencyClass::LatencyLow`. Maybe reduce the name to only `Low`